errors/errors.js

```diff
@@ -0,0 +1,21 @@
+exports.handleCustomErrors = (err, req, res, next)=> {
+    if(err.status===400) {
+        res.status(err.status).send({msg: err.msg})
+    }
+    else{
+        next(err)
+    }
+}
+
+
+
+exports.handlePsqErrors=(err, req, res, next)=>{
```

Comment:
This is what should be handling psql errors.

However at this point you can't possibly have a psql error and this should not be here.